modules/social_features/social_user/social_user.install

```diff
@@ -195,3 +195,13 @@ function social_user_update_8809() {
   // The administrator should be able to assign all roles.
   user_role_grant_permissions('administrator', ['assign all roles']);
 }
+
+/**
+ * Update permissions for the sitemanager role.
+ */
+function social_user_update_8810() {
+  // Grant the sitemanagers new permissions.
+  user_role_grant_permissions('sitemanager', [
+    'view blocked user',
```

Comment:
@dmytriikaiun please also make sure that new permission will be added on hook_install, so add it to the `_social_user_get_permissions()`.